client/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "TZ=Asia/Singapore yarn run jest --coverage",
     "testci": "TZ=Asia/Singapore yarn run jest --maxWorkers=4 --coverage",
-    "build:test": "export NODE_ENV=test && yarn run build:translations && webpack -w --node-env=production",
+    "build:test": "export NODE_ENV=test && yarn run build:translations && webpack -w --node-env=none",
```

Comment:
Why not pass in `test` instead?

Reply:
(Latest webpack only allows production, development or none)